<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>details</class>
 <widget class="QDialog" name="details">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>633</width>
    <height>499</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>商品详情</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/money_774px_1207513_easyicon.net.png</normaloff>:/money_774px_1207513_easyicon.net.png</iconset>
  </property>
  <widget class="QLabel" name="picture">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>171</width>
     <height>161</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="title">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>20</y>
     <width>378</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Adobe Arabic</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>标题</string>
   </property>
  </widget>
  <widget class="QLabel" name="author">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>360</y>
     <width>379</width>
     <height>28</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Adobe Arabic</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>作者</string>
   </property>
  </widget>
  <widget class="QLabel" name="publisher">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>390</y>
     <width>379</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Adobe Arabic</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>出版社</string>
   </property>
  </widget>
  <widget class="QLabel" name="material">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>350</y>
     <width>381</width>
     <height>35</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Adobe Arabic</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>材质：</string>
   </property>
  </widget>
  <widget class="QLabel" name="brand">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>390</y>
     <width>361</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Adobe Arabic</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>品牌：</string>
   </property>
  </widget>
  <widget class="QLabel" name="clothType">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>430</y>
     <width>371</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Adobe Arabic</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>类型：</string>
   </property>
  </widget>
  <widget class="QLabel" name="originalPlace">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>410</y>
     <width>371</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Adobe Arabic</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>生产地：</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>460</y>
     <width>236</width>
     <height>17</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <item>
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>商品编号：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="id">
      <property name="text">
       <string>1234356123671231238</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget2">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>320</y>
     <width>371</width>
     <height>23</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_8">
    <item>
     <widget class="QLabel" name="voltage">
      <property name="font">
       <font>
        <family>Adobe Arabic</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>额定电压：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="power">
      <property name="font">
       <font>
        <family>Adobe Arabic</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>额定功率：</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPlainTextEdit" name="description">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>240</y>
     <width>221</width>
     <height>191</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>100</y>
     <width>374</width>
     <height>198</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_9">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>价格：</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="price">
              <property name="font">
               <font>
                <family>Adobe Arabic</family>
                <pointsize>22</pointsize>
               </font>
              </property>
              <property name="text">
               <string>price          </string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QLabel" name="label_4">
              <property name="text">
               <string>店铺所在地：</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="place">
              <property name="font">
               <font>
                <family>Adobe Arabic</family>
                <pointsize>11</pointsize>
               </font>
              </property>
              <property name="text">
               <string>地点</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_10">
          <item>
           <widget class="QLabel" name="discountLabel">
            <property name="text">
             <string>促销：</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="discount">
            <property name="font">
             <font>
              <family>Adobe Arabic</family>
              <pointsize>18</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">color: rgb(255, 0, 54);</string>
            </property>
            <property name="text">
             <string>满减                                          </string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_11">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>数量：</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="itemNUM">
            <property name="minimum">
             <number>1</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>剩余数量：</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="number">
            <property name="font">
             <font>
              <family>Adobe Arabic</family>
              <pointsize>22</pointsize>
             </font>
            </property>
            <property name="text">
             <string>123</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPushButton" name="addToTrolley">
      <property name="font">
       <font>
        <family>Adobe Arabic</family>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 237, 237);
color: rgb(255, 0, 54);</string>
      </property>
      <property name="text">
       <string>加入购物车</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
